frontend/packages/console-shared/src/components/dynamic-form/utils.ts

```diff
@@ -23,6 +26,35 @@ export const getSchemaErrors = (schema: JSONSchema6): SchemaError[] => {
   ];
 };
 
+// Determine if a schema will produce no form fields.
+export const hasNoFields = (jsonSchema: JSONSchema6 = {}, uiSchema: UiSchema = {}): boolean => {
```

Comment:
This is probably not worth changing at this point, but I have a slight preference for `hasFields` and flipping the logic. Then you don't have to think about the negative in the function name.